src/styles/sidebar/components/markdown-editor.scss

```diff
@@ -4,6 +4,13 @@
 @use "../../mixins/utils";
 @use "../../variables" as var;
 
+.markdown-editor {
+  /* Reset line-height to avoid having extra gaps/vertical spacing in the
+     element's container
+  */
+  line-height: 1;
```

Comment:
Can you clarify what was happening without this?

Reply:
Without the `line-height` reset, there is extra vertical space (it looks like padding or margin but is not) just beneath the textarea, making the vertical rhythm look wonky (i.e. too much space between textarea and tag editor tag list).